src/main/java/com/onevizion/scmdb/facade/DbScriptFacade.java

```diff
@@ -42,15 +42,14 @@ public void init() {
     }
 
     public List<SqlScript> getNewScripts() {
-        Map<String, SqlScript> savedScripts = sqlScriptDaoOra.readMap();
-
+        Map<String, SqlScript> dbScripts = sqlScriptDaoOra.loadDbScriptsMapCached();
```

Comment:
It is confusing that `text` property is present in `SqlScript` but not populated in db calls, remove it from `SqlScript`

Reply:
No that property used in db calls particularly in DbScriptDaoOra.readMapByIds() and to get text in the rollbacks handle.

Reply:
Yes and it is wrong implementation, because now `readMapByIds()` and `readMapCached()` will return not equals `SqlScript` objects for the same record in db. So every developer have to keep in mind that `readMapCached()` returns `SqlScript` with `text` and `readMapByIds()` without, or have to check each time query for each ado call.
You need completely remove `text` from `SqlScript` and create method `DbScriptDaoOra.readTextByIds()` and use where it required

Reply:
You need to:
1. update all SQL queries in `DbScriptDaoOra` to provide a list of columns explicitly instead of using `*`
2. remove the `text` column from all SQL queries
3. completely remove the `text` field from `SqlScript`
4. create a new method `Map<Long, String> readTextMapByIds(Set<Long> ids)` to `DbScriptDaoOra` which will return `Map<db_script_id,text>`
5. use a new method to get `text` instead of calling `SqlScript.getText()`